### PR TITLE
Migrating contain matcher

### LINKED ISSUE #39: Rewrite `contain` as matcher

### PR DESCRIPTION
**Pull request by tamar-ot**
Closes #39 

Before submitting this PR, please make sure:

- [x] My code follows the style guidelines of this project
- [ ] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Type of change

- [ ] Bug fix *(non-breaking change which fixes an issue)*
- [x] New feature *(non-breaking change which adds functionality)*
- [ ] Breaking change *(fix or feature that would cause existing functionality to not work as expected)*
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by tamar-ot** [comment]
'test_method_chaining' get failed, contain matcher definition should be edit.

**Comment by bdsoha** [comment]
Seems like we can also remove `expycted.internals.utils.assertion`.

**Comment by tamar-ot** [comment]
right, will be remove.

**Comment by tamar-ot** [comment]
there is still using of this 'assertion' in  expycted.internals.filesystem. after I will edit this code Its could be remove.

File: test/unit/matchers/test_contained_in_matcher.py [hunk 1]
```diff
@@ -51,3 +52,19 @@ def test_not_matches(expectation):
...
 
     assert matcher(expectation.expected) is False
+
+
+@parametrize_expectation(
+    [
+        ("hello2", 2, "int in str"),
+        stubs.TRUE_STR_EQUIVALENT,
+        stubs.COPY_OBJECT,
+        stubs.SAME_OBJECT,
```
Comment (bdsoha) [inline test/unit/matchers/test_contained_in_matcher.py]: Use these objects instead:

https://github.com/bdsoha/expycted/blob/641a093067900912833ff71ccc4ae5830a2426c2/test/helpers/stubs.py#L70-L71

**Review by bdsoha — CHANGES_REQUESTED** [review]

**Review by bdsoha — APPROVED** [review]

**Comment by bdsoha** [comment]
LGTM

**Pull request merged**